client-sqldelight-gradle-plugin/src/test/projects/happyPath/lib/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import app.cash.backfila.client.sqldelight.plugin.SqlDelightRecordSource
+
+plugins {
+  id("app.cash.backfila.client.sqldelight")
+  id("app.cash.sqldelight")
+  kotlin("jvm")
+}
+
+sqldelight {
+  databases {
+    create("HockeyDataDatabase") {
+      packageName.set("app.cash.backfila.client.sqldelight.hockeydata")
+      dialect(libs.sqldelightMysqlDialect)
+      srcDirs.setFrom(listOf("resources/migrations"))
+      deriveSchemaFromMigrations.set(true)
+      migrationOutputDirectory.set(file("$buildDir/resources/main/migrations"))
+      verifyMigrations.set(true)
+    }
+  }
+}
+
+backfilaSqlDelight {
+  addRecordSource(
+    name = "hockeyPlayerOrigin",
+    database = "app.cash.backfila.client.sqldelight.hockeydata.HockeyDataDatabase",
+    tableName = "hockeyPlayer",
+    keyName = "player_number",
+    keyType = "kotlin.Int",
+    recordColumns = "*",
```

Comment:
Where is this handled? Are comma syntax for multiple columns handled?

Found it, looks like it supports full SQL syntax which is good. May need to note that in the documentation.

Reply:
Yes very good point.

Reply:
I've added some now. When I have it integrated in the test I will have to check that the docs are actually useful though.